src/corva/models/scheduled/initial.py

```diff
@@ -0,0 +1,8 @@
+from corva.models.base import CorvaBaseEvent
+from corva.models.scheduled.scheduler_type import SchedulerType
+
+
+class InitialScheduledEvent(CorvaBaseEvent):
```

Comment:
What's the purpose here, vs RawScheduledEvent?

Reply:
We don't know which subtype of `RawScheduledEvent` to use without `InitialScheduledEvent.scheduler_type` field

Reply:
Actually, yes, we can delete InitialScheduledEvent